modules/2d_inference_3d_volume.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install monai
</pre><p>Could you please help change to:</p><pre class="ql-syntax" spellcheck="false">!python -c "import monai" || pip install -q "monai-weekly"
</pre><p>Refer to:</p><p>https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unet_segmentation_3d_ignite.ipynb</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>